flake.nix

```diff
@@ -65,6 +65,8 @@
             treefmt
             mdl
 
+            podman
```

Comment:
This is not required, these instructions are not for developers and installing Podman here will not correctly configure it anyway.